client/html/layouts/basket/product/body-default.html

```diff
@@ -0,0 +1,114 @@
+*
+* In Model-View-Controller (MVC) applications, the controller contains the methods
+* that create parts of the output displayed in the generated HTML page. Controller
+* names are usually alpha-numeric.
+*
+* @param string Name of the controller
+* @since 2014.03
+* @category Developer
+* @see client/html/basket/standard/url/target
+* @see client/html/basket/standard/url/action
+* @see client/html/basket/standard/url/config
+*/
+$basketController = $this->config( 'client/html/basket/standard/url/controller', 'basket' );
+
+/** client/html/basket/standard/url/action
+* Name of the action that should create the output
+*
+* In Model-View-Controller (MVC) applications, actions are the methods of a
+* controller that create parts of the output displayed in the generated HTML page.
+* Action names are usually alpha-numeric.
+*
+* @param string Name of the action
+* @since 2014.03
+* @category Developer
+* @see client/html/basket/standard/url/target
+* @see client/html/basket/standard/url/controller
+* @see client/html/basket/standard/url/config
+*/
+$basketAction = $this->config( 'client/html/basket/standard/url/action', 'index' );
+
+/** client/html/basket/standard/url/config
+* Associative list of configuration options used for generating the URL
+*
+* You can specify additional options as key/value pairs used when generating
+* the URLs, like
+*
+*  client/html/
+<clientname>/url/config = array( 'absoluteUri' => true )
+	*
+	* The available key/value pairs depend on the application that embeds the e-commerce
+	* framework. This is because the infrastructure of the application is used for
+	* generating the URLs. The full list of available config options is referenced
+	* in the "see also" section of this page.
+	*
+	* @param string Associative list of configuration options
+	* @since 2014.03
+	* @category Developer
+	* @see client/html/basket/standard/url/target
+	* @see client/html/basket/standard/url/controller
+	* @see client/html/basket/standard/url/action
+	* @see client/html/url/config
+	*/
+	$basketConfig = $this->config( 'client/html/basket/standard/url/config', array() );
+
+	$enc = $this->encoder();
+
+	?>
+	<div class="basket">
+		<button href="#" data-dropdown="drop2" aria-controls="drop2" aria-expanded="false" class="button dropdown"><i
+				class="fa fa-shopping-cart"></i><?php echo $enc->html( $this->translate( 'client/html', 'My basket' ),
+			$enc::TRUST ); ?>
+		</button>
+		<ul id="drop2" data-dropdown-content class="f-dropdown" aria-hidden="true">
+			<?php echo $this->get( 'miniBody' ); ?>
+			<!--<li><span>Bioderma ABC Derm</span><span>2</span><span>€19,80</span></li>-->
+			<!--<div class="clearfix"></div>-->
+			<!--<li><span>Bioderma ABC Derm</span><span>2</span><span>€19,80</span></li>-->
+			<!--<div class="last clearfix"></div>-->
+			<div class="clearfix"></div>
+			<li>
+				<div class="checkoutbtn">
+					<a href="/basket" class="btn right mt-15 mb-30">Afrekenen</a>
```

Comment:
Don't use fixed URLs, they will work only in your setup. You can generate URL with $this->url(...) and there are numerous examples in the templates for this. Also, you should use $this->translate(...) so strings can be translated to other languages:
<?php echo $enc->html( $this->translate( 'client/html', 'Checkout' ), $enc::TRUST ); ?>